src/app/api/[chain]/schedule/route.ts

```diff
@@ -25,7 +23,7 @@ export const GET = async (req: NextRequest, ctx: AppRouteHandlerFnContext) => {
     const locationIds = queryParams.getAll("locationId");
     const chainIdentifier = chainIdentifierFromContext(ctx);
     if (chainIdentifier === null) return respondNotFound();
-    const chain: RezervoChain = await getChain(chainIdentifier);
+    const chain = await fetchChain(chainIdentifier);
```

Comment:
Not introduced here, but this fetch is pointless. You can just use `chainIdentifier` directly instead of `chain.profile.identifier`.

Reply:
Fixed in https://github.com/mathiazom/rezervo-web/pull/119/commits/e42296305ba6cac806ca7dc845123125cb73cc3d